diff --git a/uppbox/Scripts/CLANGMK.bm b/uppbox/Scripts/CLANGMK.bm
index 3d5da87c2..73aeb8ad1 100644
--- a/uppbox/Scripts/CLANGMK.bm
+++ b/uppbox/Scripts/CLANGMK.bm
@@ -1,6 +1,6 @@
 BUILDER = "CLANG";
 COMPILER = "clang++";
-COMMON_CPP_OPTIONS = "-std=c++14 -Wall -Wno-logical-op-parentheses";
+COMMON_CPP_OPTIONS = "-std=c++17 -Wall -Wno-logical-op-parentheses";
 COMMON_C_OPTIONS = "";
 COMMON_FLAGS = "";
 DEBUG_INFO = "2";
diff --git a/uppbox/Scripts/GCCMK.bm b/uppbox/Scripts/GCCMK.bm
index 5fcef437e..dba91ecc5 100644
--- a/uppbox/Scripts/GCCMK.bm
+++ b/uppbox/Scripts/GCCMK.bm
@@ -1,5 +1,5 @@
 BUILDER = "GCC";
-COMMON_CPP_OPTIONS = "-std=c++14";
+COMMON_CPP_OPTIONS = "-std=c++17";
 DEBUG_INFO = "2";
 DEBUG_BLITZ = "1";
 DEBUG_LINKMODE = "1";
