diff --git a/uppsrc/ide/Core/Assembly.cpp b/uppsrc/ide/Core/Assembly.cpp
index 3e3e87180..38f465a60 100644
--- a/uppsrc/ide/Core/Assembly.cpp
+++ b/uppsrc/ide/Core/Assembly.cpp
@@ -156,6 +156,7 @@ bool SaveVars(const char *name)
 String GetDefaultUppOut()
 {
 	String out;
+#ifndef FLATPAK
 	String p = GetExeFolder();
 	while(p.GetCount() > 1 && DirectoryExists(p)) {
 		String h = AppendFileName(p, ".cache");
@@ -165,7 +166,8 @@ String GetDefaultUppOut()
 		}
 		p = GetFileFolder(p);
 	}
-	
+#endif
+
 	out = Nvl(out, GetHomeDirFile(".cache")) + "/upp.out";
 	
 	RealizeDirectory(out);
